src/styles/blocks/card.scss

```
@@ -0,0 +1,142 @@
+@import "../utils/variables.scss";
+
+.catalog {
+  padding-top: 50px;
+  display: grid;
+  grid-template-columns: repeat($column-count, $width-card);
+  grid-template-rows: repeat(2, 1fr);
+  gap: 46px 48px;
+  justify-content: center;
+
+  @media (max-width: 487px) {
+    $column-count: 1;
+    grid-template-columns: repeat($column-count, $width-card);
+  }
+
+  @media (min-width: 488px) {
+    $column-count: 2;
+    grid-template-columns: repeat($column-count, $width-card);
```

It would be better if you didn't use the variable here and simply do
```suggestion
    grid-template-columns: repeat(2, $width-card);
```